supervision/utils/video.py

```diff
@@ -60,6 +62,7 @@ def resolution_wh(self) -> tuple[int, int]:
         return self.width, self.height
 
 
+@deprecated
```

Comment:
I'd expect a bit more informative message.

```python
@deprecated(
    "`VideoSink` enum is deprecated and will be removed in "
    "`supervision-0.32.0`. Use `Video().sink` instead."
)
```

That would need to be applied to all `deprecated` decorators.